### ISSUE TITLE
Display current principle as page title

### ISSUE THREAD

**Issue by hidde** [w3c/wai-atag-report-tool#17]
Rather than having the same page title throughout all steps, have the current step as page title. (cc @nitedog)

**Comment by hidde** [comment]
After implementing this suggestion, I found the page title misses  context.

I think when it is just “UI Perceivable' it is hard to understand what that heading is for. I started a process of going through these steps, but am I still in that process or have I gone somewhere else? 

I've addressed this by adding “Your Evaluation” plus an indication of where the user is in the process (eg '2/8'), visually just before the heading (in HTML, it is part of the heading).

<img width="250" alt="small capital letters show your principle and   2 out of 8, large heading shows UI Perceivable, the name of the principle" src="https://user-images.githubusercontent.com/178782/72874454-04b4c580-3d2d-11ea-8885-efed1bd32488.png">
